crates/bevy_pbr/src/light.rs

```diff
@@ -580,17 +584,14 @@ fn cluster_space_light_aabb(
     );
     // clamp to ndc coords
     (
-        aabb_min.clamp(
-            Vec3::new(-1.0, -1.0, f32::MIN),
-            Vec3::new(1.0, 1.0, f32::MAX),
-        ),
-        aabb_max.clamp(
-            Vec3::new(-1.0, -1.0, f32::MIN),
-            Vec3::new(1.0, 1.0, f32::MAX),
-        ),
+        aabb_min.clamp(NDC_MIN, NDC_MAX),
+        aabb_max.clamp(NDC_MIN, NDC_MAX),
     )
 }
 
+const NDC_MIN: Vec3 = const_vec3!([-1.0, -1.0, f32::MIN]);
+const NDC_MAX: Vec3 = const_vec3!([1.0, 1.0, f32::MAX]);
```

Comment:
This is a misnomer. ndc min/max depth values are 0.0 and 1.0 as anything outside will be clipped. I’ll have a closer look what this bit of code is doing to see whether the values should be changed or whether there’s a more accurate name that can be used.

Reply:
I reordered the code to not need to clamp on a Vec3, but do it on the Vec2. The clamp on z was meaningless anyway as it was between `f32::MIN` and `f32::MAX`. This seem to be even a tad bit faster to execute 🎉